test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1103,13 +1103,178 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
+            autoEncryption: {
+              keyVaultNamespace,
+              kmsProviders: { local: { key: LOCAL_KEY } },
+              schemaMap: { dataNamespace: externalSchema },
+              // Configure the following `extraOptions`
+              // {
+              //    "mongocryptdBypassSpawn": true
+              //    "mongocryptdURI": "mongodb://localhost:27021/db?serverSelectionTimeoutMS=1000",
+              //    "mongocryptdSpawnArgs": [ "--pidfilepath=bypass-spawning-mongocryptd.pid", "--port=27021"]
+              // }
+              extraOptions: {
+                mongocryptdBypassSpawn: true,
+                mongocryptdURI: 'mongodb://localhost:27021/db?serverSelectionTimeoutMS=1000',
+                mongocryptdSpawnArgs: [
+                  '--pidfilepath=bypass-spawning-mongocryptd.pid',
+                  '--port=27021'
+                ]
+              }
+            }
+          }
+        );
+      });
+
+      beforeEach('precondition: the shared library must NOT be loaded', function () {
+        const { cryptSharedLibPath } = getEncryptExtraOptions();
+        if (cryptSharedLibPath) {
+          this.currentTest.skipReason =
+            'test requires that the shared library NOT is present, but CRYPT_SHARED_LIB_PATH is set.';
+          this.skip();
+        }
+        // the presence of the shared library can only be reliably determine after
+        // libmongocrypt has been initialized, and can be detected with the
+        // cryptSharedLibVersionInfo getter on the autoEncrypter.
+        expect(!!clientEncrypted.autoEncrypter.cryptSharedLibVersionInfo).to.be.false;
+      });
+
+      afterEach(async function () {
+        await clientEncrypted?.close();
+      });
+
+      it('does not spawn mongocryptd', metadata, async function () {
+        // Use client_encrypted to insert the document {"encrypted": "test"} into db.coll.
+        // Expect a server selection error propagated from the internal MongoClient failing to connect to mongocryptd on port 27021.
+        const insertError = await clientEncrypted
+          .db(dataDbName)
+          .collection(dataCollName)
+          .insertOne({ encrypted: 'test' })
+          .catch(e => e);
+
+        expect(insertError)
+          .to.be.instanceOf(Error)
+          .to.have.property('name', 'MongoServerSelectionError');
 
-    it.skip('Via bypassAutoEncryption', () => {}).skipReason =
-      'TODO(NODE-2422): Implement "Bypass spawning mongocryptd" tests';
+        expect(insertError).to.match(/connect ECONNREFUSED 127.0.0.1:27021/);
+
+        expect(insertError).to.be.instanceOf(
+          MongoServerSelectionError,
+          `
+
+TODO(NODE-5283): The error thrown in this test fails an instanceof check with MongoServerSelectionError.
+  This should change after NODE-5283.  If this assertion is failing, then the test
+  should be updated to reflect that the error thrown is now a server selection error.
+
+`
+        );
```

Comment:
This assertion is not necessary, but will fail once NODE-5283 is completed and will remind us to update this test.

Reply:
so should it be inverted above? `not.to.be.instanceOf`?